File: PreferenceSample/app/src/main/java/com/example/iyemon018/preferencesample/pref/NotificationPrefModel.java
package com.example.iyemon018.preferencesample.pref;

import com.os.operando.garum.annotations.Pref;
import com.os.operando.garum.annotations.PrefKey;
import com.os.operando.garum.models.PrefModel;

@Pref
public final class NotificationPrefModel extends PrefModel {
    
    @PrefKey("notifications_new_message")
    public boolean notificationsNewMessage;
    
    @PrefKey("notifications_new_message_ringtone")
    public String notificationsNewMessageRingtone;
    
    @PrefKey("notifications_new_message_vibrate")
    public boolean notificationsNewMessageVibrate;
}

File: PreferenceSample/app/src/main/java/com/example/iyemon018/preferencesample/NotificationPrefModel.kt
package com.example.iyemon018.preferencesample

import com.os.operando.garum.annotations.Pref
import com.os.operando.garum.annotations.PrefKey
import com.os.operando.garum.models.PrefModel

@Pref
class NotificationPrefModel : PrefModel() {

    @PrefKey("notifications_new_message")
    public var notificationsNewMessage: Boolean = false

    @PrefKey("notifications_new_message_ringtone")
    public var notificationsNewMessageRingtone: String = ""

    @PrefKey("notifications_new_message_vibrate")
    public var notificationsNewMessageVibrate: Boolean = false
}
